_slides/01-nkatangoor.md

```diff
@@ -4,3 +4,4 @@ title: "Welcome to our slide deck!"
 ---
 
 Use the right arrow to begin!
```

Comment:
@nkatangoor Thank you for your comment, can you add more content?

Reply:
@evgenyrahman Sure will add more!